src/configs.rs

```diff
@@ -136,14 +136,16 @@ impl Config {
             search_dirs
                 .iter()
                 .map(|search_dir| {
-                    let expanded_path = shellexpand::full(&search_dir.path.to_string_lossy())
+                    let original_path = search_dir.path.to_string_lossy();
+                    let expanded_path = shellexpand::full(&original_path)
                         .change_context(ConfigError::IoError)
-                        .unwrap()
+                        .as_ref()
+                        .unwrap_or(&original_path)
```

Comment:
Wouldn't it be better to filter it out instead of adding a non existing search path?

Reply:
You're right. Chnaged it to filter seach_dirs and also search_paths.

Reply:
Lgtm